ceto/ais_adapter.py

```diff
@@ -405,7 +409,7 @@ def guesstimate_vessel_data(
 
     # Vessel size (DWT)
     vessel_size = _guesstimate_vessel_size_as_DeadWeightTonnage(
```

Comment:
I did not notice the name of the functions before. I am not comfortable with seeing a mix of camel case and snake case. Snakes and camels do not like each other, right? What about `_guesstimate_vessel_size_as_deadweight_tonnage` or `_guesstimate_vessel_size_as_dwt`?

Reply:
Yep, you're right, corrected.